apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -278,14 +278,26 @@ public async Task RollbackAsync(RollbackRequest request, CallSettings callSettin
         /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
         /// <returns>A task representing the asynchronous operation. When the task completes, the result is the response from the RPC.</returns>
         public Task<PartitionResponse> PartitionQueryAsync(PartitionQueryRequest request, CallSettings callSettings)
+            => PartitionReadOrQueryAsync(request, callSettings);
+
+        /// <summary>
+        /// Executes a PartitionRead RPC asynchronously.
+        /// </summary>
+        /// <param name="request">The partitioning request. Must not be null. The request will be modified with session details
+        /// from this object.</param>
+        /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
+        /// <returns>A task representing the asynchronous operation. When the task completes, the result is the response from the RPC.</returns>
+        public Task<PartitionResponse> PartitionReadOrQueryAsync(IReadOrQueryPartitionRequest request, CallSettings callSettings)
```

Comment:
Yep, I don't think we should make this public. This interface (or alternative) should remain an implementation detail.